api/handler/default.go

```diff
@@ -249,6 +249,7 @@ func (m ArcDefaultHandler) POSTTransactions(ctx echo.Context, params api.POSTTra
 			return ctx.JSON(int(api.ErrStatusBadRequest), e)
 		}
 
+		txMap := make(map[string]*bt.Tx)
```

Comment:
Why do you create a map of the whole transaction, instead of just the sizing, which is the only thing it is being used for?